llvm/tools/llvm-exegesis/lib/X86/Target.cpp

```diff
@@ -1009,6 +1011,10 @@ static std::vector<MCInst> loadImmediateSegmentRegister(unsigned Reg,
 #endif // defined(__x86_64__) && defined(__linux__)
 }
 
+unsigned ExegesisX86Target::findRegisterByName(const StringRef RegName) const {
```

Comment:
Can we just have a default implementation that returns `MCRegister::NoRegister` instead of having a function for every target?

Reply:
Addressed